### PR TITLE
✅ Add e2e tests for `amp-tiktok`

### PR DESCRIPTION
**Pull request by rebeccanthomas**
Add e2e tests for the `amp-tiktok` component. 
This test verifies that the tiktok component resizes as expected, which is when the given height is less than 578px, the component will resize to be the full height of the component

I2I: #32219

### CODE REVIEW

**Comment by rebeccanthomas** [comment]
This test is currently skipped because it fails, for some reason, despite the generous wait time given to allow for the component to resize in the browser, it seems that the iframe's height never actually changes. 

Does anyone know why this might be the case?

**Review by kristoferbaxter — COMMENTED** [review]
Will take a look on Monday to see if I can help with the test failure.

Just happened to notice this when on my phone.

File: extensions/amp-tiktok/0.1/test-e2e/test-amp-tiktok.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    describe('tiktok component with video source link as src', function () {
+      it('resizes if the tiktok element is smaller than the height of the tiktok player', async () => {
+        const iframe = await selectContentIframe('tiktok-2');
+        const initialIframeRect = await controller.getElementRect(iframe);
+        const initialHeight = await initialIframeRect.height;
+        // Expect iframe to initialize to default height
+        await expect(initialHeight).to.equal(500);
+
+        // 1500ms chosen to allow sufficient time for tiktok to recieve the new height message and resize.
+        await new Promise((resolve) => {
```
Comment (alanorozco) [inline extensions/amp-tiktok/0.1/test-e2e/test-amp-tiktok.js:61]: - Do we wait in any other e2e tests?

- Corollary: Do we have e2e tests for any other iframes to external resources? I don't think we do.

Comment (rebeccanthomas) [reply extensions/amp-tiktok/0.1/test-e2e/test-amp-tiktok.js:61]: I don't believe we do, This test was initially set up to verify the messages from TikTok were being received by the iframe, however I agree that that isn't a good approach. 

In this case the component is likely better served with just unit tests.

**Pull request closed without merging**